[Docs] - Updates color usage page with accent color content (#6280)
grommet/hpe-design-system: apps/docs/src/pages/design-tokens/color-usage.mdx

| <ColorPreview datum={{value: 'background-info'}} />            | info     | Informational content.                                                        |
| <ColorPreview datum={{value: 'background-ok'}} />              | ok       | Success or everything is operating as expected.                               |
| <ColorPreview datum={{value: 'text-onStrong'}} />              | onStrong | For content (text and icons) that sit on strong backgrounds.                  |

> | <ColorPreview datum={{value: '#6B5CF6'}} />                    | accent   | For content that requires differentiation. Carries no semantic meaning.       |

Reviewer: @julauxen What do you think we make the point first that (1) there is no inherent semantic meaning, then (2) a possible use-case?

```suggestion
| <ColorPreview datum={{value: '#6B5CF6'}} />                    | accent   | Carries no semantic meaning. Use for differentiation or emphasis.       |
```

Author: I updated
